designs/administrator/olm/sort-by-recently-updated/sort-by-recently-updated.md

```diff
@@ -0,0 +1,48 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Sort by Provider update on OperatorHub
+
+When viewing the list of Operators available for install on OperatorHub, the users are interested in which Operators were updated most recently. OperatorHub should support sorting by the most recently updated Operator.
+
+## Sorting on OperatorHub
+
+The Sort by menu and View icons allow the user to now further customize their sorting experience on Operator Hub.
+
+![Sort-by-on-OperatorHub](img/1-filter-on-operator-hub.jpg)
+
+## User selects “Provider update” on menu
```

Comment:
Would "Last updated" work better? The provider for a lot of these is Red Hat or the company rather than the operator itself.

Reply:
Yes, we tried to float last updated, recently updated, and date version released. Ux and product found "Last updated" confusing because we use Last updated in a different capacity to represent when the user has updated an installed operator in the list of all operators later in the flow. We consulted with Abi and landed on Provider update. We'll see what other stakeholders have to say, but I think this is the most accurate wording I've heard to date.

Reply:
Sounds good!! Thanks for sharing the path you took to get there :)